channel/src/main/java/com/synopsys/integration/alert/channel/jira/cloud/descriptor/JiraCloudDistributionUIConfig.java

```diff
@@ -67,8 +86,33 @@ public JiraCloudDistributionUIConfig() {
         ConfigField resolveWorkflow = new TextInputConfigField(JiraCloudDescriptor.KEY_RESOLVE_WORKFLOW_TRANSITION, LABEL_RESOLVE_WORKFLOW_TRANSITION, DESCRIPTION_RESOLVE_WORKFLOW_TRANSITION);
         ConfigField openWorkflow = new TextInputConfigField(JiraCloudDescriptor.KEY_OPEN_WORKFLOW_TRANSITION, LABEL_OPEN_WORKFLOW_TRANSITION, DESCRIPTION_OPEN_WORKFLOW_TRANSITION)
                                        .applyRequiredRelatedField(resolveWorkflow.getKey());
+        ConfigField fieldMapping = new FieldMappingEndpointField(JiraCloudDescriptor.KEY_FIELD_MAPPING, LABEL_FIELD_MAPPING, DESCRIPTION_FIELD_MAPPING, "Jira Field", "Value")
+                                       .applyNewMappingTitle("Create Jira Field Mapping")
+                                       .applyValidationFunctions(this::validateFieldMapping);
+
```

Comment:
We should put this in its own field group. "Advanced" or something. I think you can apply it to `ConfigField`.